modules/core/CMakeLists.txt

```diff
@@ -145,8 +145,7 @@ ocv_create_module(${extra_libs})
 
 ocv_target_link_libraries(${the_module} PRIVATE
     "${ZLIB_LIBRARIES}" "${OPENCL_LIBRARIES}" "${VA_LIBRARIES}"
-    "${OPENGL_LIBRARIES}"
-    "${GLX_LIBRARIES}"
```

Comment:
It looks like `GLX_LIBRARIES` is rudiment of some old CMake or cmake script. Modern CMake does not set it and it's always empty on Linux.

Reply:
Cool. Thanks!